### PR TITLE
handwired/videowriter: oleg keymap update, via keymap

### PR DESCRIPTION
**Pull request by DmNosachev**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/videowriter/config.h [hunk 1]
```diff
@@ -20,12 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 /* USB Device descriptor parameter */
-#define VENDOR_ID 0xFEED
-#define PRODUCT_ID 0x0000
+#define VENDOR_ID 0x515A // "QZ"
+#define PRODUCT_ID 0x5657 // "VW"
 #define DEVICE_VER 0x0001
 #define MANUFACTURER Magnavox
-#define PRODUCT Videowriter
-#define DESCRIPTION Keyboard
+#define PRODUCT Magnavox Videowriter
```
Comment (drashna) [inline keyboards/handwired/videowriter/config.h:27]: https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> don’t repeat MANUFACTURER in the PRODUCT value

```suggestion
#define PRODUCT Videowriter
```

File: keyboards/handwired/videowriter/config.h [hunk 2]
```diff
@@ -20,12 +20,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+#define DESCRIPTION Magnavox Videowriter
```
Comment (drashna) [inline keyboards/handwired/videowriter/config.h:28]: This Setting is no longer used, and should be removed. 
```suggestion
```

File: keyboards/handwired/videowriter/keymaps/via/rules.mk [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# this particular convertion project uses Pro Micro with DFU bootloader instead of Caterina
+BOOTLOADER = lufa-dfu
```
Comment (drashna) [inline keyboards/handwired/videowriter/keymaps/via/rules.mk:2]: Usually, this is atmel-dfu, or qmk-dfu.  lufa-dfu is rather uncommon.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/videowriter/keymaps/via/rules.mk (hunk 3 above)
Comment (DmNosachev) [reply keyboards/handwired/videowriter/keymaps/via/rules.mk:2]: It's actually a lufa-dfu (from my fork, adapted for Pro Micro, as I use lot of them, not only for QMK). I forgot that this code will be used by everyone, so I'll change the bootloader setting to default Caterina

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**